tests/end-to-end/director/request_test.go

```diff
@@ -0,0 +1,17 @@
+package director
```

Comment:
Not sure about the name of this file, something like `fixtures_test.go` would fit better IMO but there's already `fixtures.go` so that would look as if those two were related. Also, why only this one mutation has a fixture? We should discuss it.